pycheribuild/projects/cross/cheribsd.py

```diff
@@ -604,8 +620,14 @@ def kernel_make_args_for_config(self, kernconf: str, extra_make_args) -> MakeOpt
         if self.compiling_for_mips(include_purecap=True):
             # Don't build kernel modules for MIPS
             kernel_options.set(NO_MODULES="yes")
-        if self.compiling_for_riscv(include_purecap=True):
+        elif self.compiling_for_riscv(include_purecap=True):
             kernel_options.set_with_options(CTF=False)  # FIXME: restore once debugged
+        elif self.crosscompile_target.is_hybrid_or_purecap_cheri([CPUArchitecture.AARCH64]):
+            # Disable CTF for now to avoid the following errors:
+            # ERROR: cam_periph.c: die 25130: unknown base type encoding 0xffffffffffffffa1
+            kernel_options.set_with_options(CTF=False)
+            # Only build SMB module (since e.g. Linux module is broken)
+            kernel_options.set(MODULES_OVERRIDE="smbfs libiconv libmchain")
```

Comment:
I'm not sure how useful these module are. `smbfs` only talks an old version of SMB with known security problems that I seem to remember was removed (or will soon be removed) from Samba.

For now I'd just disable all modules.

Reply:
I think we still use it in CI for copying files out, though without QEMU we might need a different approach anyway.